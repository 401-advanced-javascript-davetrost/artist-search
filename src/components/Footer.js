import React from 'react';

export default function Footer() {

  return (
    <footer>
      <p>Â©Abbey, Angela, Pat, and Dave</p>
    </footer>
  );
}
