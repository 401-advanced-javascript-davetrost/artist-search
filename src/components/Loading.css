.Loading {
  width: 100vw;
  display: flex;
  justify-content: center;
  align-items: center;

  img {
    height: 400px;
  }
  
}