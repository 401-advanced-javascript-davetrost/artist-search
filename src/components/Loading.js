import React from 'react';
import styles from './Loading.css';

export default function Loading() {
  return (
    <div className={styles.Loading}>
      <img src="/src/assets/loading.gif" />
    </div>
  );
}
