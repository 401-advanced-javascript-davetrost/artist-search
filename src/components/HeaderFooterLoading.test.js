import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';
import Footer from './Footer';
import Loading from './Loading';

describe('Header component', () => {
  it('renders Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('Footer component', () => {
  it('renders Footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('Loading component', () => {
  it('renders Loading', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
