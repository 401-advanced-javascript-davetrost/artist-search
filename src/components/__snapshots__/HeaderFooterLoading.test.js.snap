// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders Footer 1`] = `
<footer
  className="Footer"
>
  <p>
    Â© Abbey, Angela, Pat, and Dave
  </p>
</footer>
`;

exports[`Header component renders Header 1`] = `
<header
  className="Header"
>
  <h1>
    Artist Search
  </h1>
  <nav>
    <Link
      to="/"
    >
      Home
    </Link>
  </nav>
</header>
`;

exports[`Loading component renders Loading 1`] = `
<div
  className="Loading"
>
  <img
    src="/src/assets/loading.gif"
  />
</div>
`;
